### ISSUE TITLE
Navigation issues on Android TV

### ISSUE THREAD

**Issue by gkeegan** [TeamNewPipe/NewPipe#4004]
### Version
- 0.19.8

### Expected behavior
Android TV navigation works well and doesn't require workarounds in order to click certain buttons.

### Actual behaviour
Navigating to get to certain buttons (in particular the different feeds on the subscription page and the refresh feed button) is very wonky and requires setting the cursor in certain positions in order to navigate to those buttons. 

The video shows these issues quite well.

### Screenshots/Screen recordings
https://streamable.com/smm3k5

I'm waiting to peel the stickers off the TV once I get it on the cabinet, don't hurt me lol.

**Comment by Stypox** [comment]
@Alexander-- would you mind taking a look?

**Comment by Alexander--** [comment]
@Stypox I didn't watch the video (@gkeegan could you upload it to youtube?) but I suspect, that everything works as designed.

The subscription page used to be better when I initially worked on it, but ever since it  went through round of beautification in #2309 it have gotten a lot worse. The entire subscription group screen has become a single massive RecyclerView with a GridLayoutManager. With all associated focus handling bugs.

Normally, it is possible to work around most focus issues by manually assigning nextFocusXxx in XML layout files. But in case of action bar icons and RecyclerViews that does not work, because there is no coherent layout file... When I fixed up a comment feed, I wrote a custom LinerLayoutManager, — SuperScrollLayoutManager — that dealt with most of it's issues. Unfortunately, writing something like that for GridLayoutManager proved significantly harder. Possibly multiple days worth of work.

If navigation works at all on that screen, I'd consider that great already.

**Comment by gkeegan** [comment]
@Alexander-- yes it does feel like a grid issue, since a lot of the behavior feels like I'm dealing with a grid of different sized boxes.

I think an interim solution would be to make the buttons for feeds the same size as each subscription so it is more obvious how to navigate to it. Or, if it is possible, changing the starting position of the cursor to somewhere different.

And I cannot upload to YouTube sorry. I could use another video host that doesn't require sign in if streamable doesn't work right.

**Comment by jstaro** [comment]
How _do_ you actually refresh the subscription list? I seem completely unable to navigate to that button.

**Comment by christopher-wilson** [comment]
@jstaro If you are still having an issue, maybe try navigating to your middle tab, then pressing down.
